cdroot.overlay/isolinux/menu.cfg

```diff
@@ -2,13 +2,13 @@ default cold-offline
 label cold-offline
   menu label cold-offline: create wallet, sign transactions
   kernel /casper/vmlinuz
-  append boot=casper initrd=/casper/initrd.gz root=/dev/ram rw showmounts debug toram bitkey=cold-offline --
+  append boot=casper initrd=/casper/initrd.gz root=/dev/ram rw showmounts toram bitkey=cold-offline --
```

Comment:
Is there a reason why you removed the debug option? I suspect that it was explicitly set to ensure that you could easily audit the boot it for security purposes.